serverdock/Dockerfile

```diff
@@ -4,24 +4,18 @@ MAINTAINER "dror@tabookey.com"
 RUN	apt-get update && \
 	apt-get install -y git nodejs npm
 
-RUN npm install -g n
+RUN npm install -g n yarn
 RUN n stable
 
 RUN npm install -g ganache-cli@6.4.3 truffle
-RUN npm install truffle-hdwallet-provider
+RUN echo '{}' > package.json
 RUN npm install web3@1.0.0-beta.37
-RUN npm install openzeppelin-solidity@2.1.2
-RUN npm install @0x/contracts-utils@3.1.1
+#not sure why this breaks installing other packages..
+RUN rm -rf node_modules/websocket
+RUN npm install openzeppelin-solidity@2.1.2 @0x/contracts-utils@3.1.1
 
-
```

Comment:
Why aren't these files needed anymore?

Reply:
They are in use. I just add them with a single `ADD . .`